posts/tips-and-tricks-for-manually-scaling-a-global-dynamodb-table-from-an-aws-lambda.md

```diff
@@ -0,0 +1,128 @@
+---
+authors:
+- Dennis Sharpe
+tags:
+- AWS
+- Lambda
+- DynamoDB
+- Java
+date: 2020-11-16T15:52:00.000Z
+title: "Tips and Tricks for Manually Scaling a Global DynamoDB Table from an AWS Lambda"
+image:
+---
+# Objective
+Write an AWS Lambda that manually scales a global DynamoDB table
+
+# Why?
+DynamoDB tables can be set to automatically scale based on load. However, this process can take quite a while. We needed to scale up at a precise time to be ready for large batch loads.
+
+# MVP
+Scale a single (non-global) table
+- Turn off auto-scaling for the table (Provisioned)
+- Use the DynamoDB SDK v1
+- Explicitly call the SDK to set the RCU/WCU on the table
```

Comment:
I would potentially explain what "RCU/WCU" is in this case. I'd avoid acronyms/initialisms in general if possible unless they are common like "SDK". If you want you can throw links in to these concepts too to give the reader a head start.